src/desktop/apps/artwork/components/auction/view.coffee

```diff
@@ -65,10 +67,23 @@ module.exports = class ArtworkAuctionView extends Backbone.View
     $target = $(e.currentTarget)
     $target.attr 'data-state', 'loading'
 
-    acquire AUCTION.artwork_id
-      .catch ->
-        $target.attr 'data-state', 'error'
-        location.reload()
+    # Show the new buy now flow if you have the lab feature enabled
+    loggedInUser = CurrentUser.orNull()
+    if loggedInUser?.hasLabFeature('New Buy Now Flow')
+      createOrder
+        artworkId: AUCTION.artwork_id
+        quantity: 1
```

Comment:
Is `quantity` needed either?

Reply:
I believe it defaults to `1` but I like being explicit (not a hard req though).

Reply:
I'll leave it then. Doesn't hurt to have it.